consensus/src/simplex/signing_scheme/mod.rs

```diff
@@ -116,6 +142,17 @@ pub trait Scheme: Clone + Debug + Send + Sync + 'static {
     /// for the given round.
     fn seed(&self, round: Round, certificate: &Self::Certificate) -> Option<Self::Seed>;
 
+    /// Returns whether per-validator fault evidence can be safely exposed.
+    ///
+    /// Schemes where individual signatures can be safely reported as fault evidence should
+    /// return `true`. Threshold schemes that are vulnerable to signature forgery attacks
```

Comment:
nit: I don't think we need to make a blanket claim about threshold here and should scope this just for specific impls